homeassistant/components/onkyo/config_flow.py

```diff
@@ -0,0 +1,246 @@
+"""Config flow for Onkyo."""
+
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+import eiscp
+import voluptuous as vol
+
+from homeassistant.config_entries import (
+    ConfigEntry,
+    ConfigFlow,
+    ConfigFlowResult,
+    OptionsFlow,
+    OptionsFlowWithConfigEntry,
+)
+from homeassistant.const import CONF_HOST, CONF_MAC, CONF_MODEL, CONF_NAME, CONF_PORT
+from homeassistant.core import callback
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.selector import ObjectSelector
+from homeassistant.util.network import is_ip_address
+
+from .const import (
+    BRAND_NAME,
+    CONF_DEVICE,
+    CONF_LEGACY_MAXIMUM_VOLUME,
+    CONF_LEGACY_RECEIVER_MAXIMUM_VOLUME,
+    CONF_MAXIMUM_VOLUME,
+    CONF_MAXIMUM_VOLUME_DEFAULT,
+    CONF_PORT_DEFAULT,
+    CONF_RECEIVER_MAXIMUM_VOLUME,
+    CONF_RECEIVER_MAXIMUM_VOLUME_DEFAULT,
+    CONF_SOURCES,
+    CONF_SOURCES_DEFAULT,
+    DOMAIN,
+    EISCP_IDENTIFIER,
+    EISCP_MODEL_NAME,
+)
+
+FlowInput = Mapping[str, Any] | None
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class OnkyoConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Onkyo config flow."""
+
+    VERSION = 2
+    MINOR_VERSION = 0
```

Comment:
Must I not provide a version for a config flow for later migrations?

Reply:
Ideally we should not build for future migrations in mind :P

But the default is VERSION = 1 and MINOR_VERSION = 1. and currently there is no entry, so it can just use the default